src/extension/app/store/app.js

```diff
@@ -129,6 +159,18 @@ export class AppStore {
    */
   @observable accessor state = STATE.INITIALIZING;
 
+  /**
+   * The bulk selection (admin mode only)
+   * @type {BulkSelection}
+   */
+  @observable accessor bulkSelection = [];
```

Comment:
wdyt about creating a `selectionStore` or `bulkStore` and moving all this logic related to bulk operations in there? Just thinking that depending on where we go with bulk in the future (maintaining lists) it could pollute app.js more than it already is.

Maybe we could only initialize this store if in the admin view as well? Not 100% sure of that tho..

Reply:
`selectionStore` could also contain an instance of appStore the same way as `siteStore` does.

Reply:
Refactored to use `bulkStore` which only gets initialized if in the admin view.